tests/openshift-postgres/namespace_test.go

```diff
@@ -0,0 +1,23 @@
+package builtin
+
+import (
+	"testing"
+
+	"github.com/projectsyn/component-keycloak/common"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	expectedNsLabels = map[string]string{
+		"SYNMonitoring": "main",
```

Comment:
I agree, but this would actually violate the test as it is not conditional hard coded. https://github.com/projectsyn/component-keycloak/blob/master/component/main.jsonnet#L13

Reply:
My comment was more regarding that we should fix this in order to be able to scrape metrics on OpenShift

Reply:
Created #205 as a follow up.